dockerfiles/theia/build.sh

```diff
@@ -10,6 +10,19 @@
 base_dir=$(cd "$(dirname "$0")"; pwd)
 . "${base_dir}"/../build.include
 
+DIR=$(cd "$(dirname "$0")"; pwd)
+LOCAL_ASSEMBLY_DIR="${DIR}"/che-theia
+
+if [ -d "${LOCAL_ASSEMBLY_DIR}" ]; then
+  rm -r "${LOCAL_ASSEMBLY_DIR}"
+fi
+
+#in mac os 'cp' cannot create destination dir, so create it first
+mkdir ${LOCAL_ASSEMBLY_DIR}
+
+echo "Copying ${base_dir}/../../extensions/extensions.yml --> ${LOCAL_ASSEMBLY_DIR}/extensions.yml"
+cp "${base_dir}/../../extensions/extensions.yml" "${LOCAL_ASSEMBLY_DIR}/"
```

Comment:
I would have used ln -s to just create symlink (and avoid copy each time) but I'm fine with the cp

Reply:
Docker cannot follow symlinks https://github.com/moby/moby/issues/1676

Reply:
Hard link then (like we did there two years ago for che-server)
https://github.com/eclipse/che/blob/5.1.0/dockerfiles/che/build.sh#L27

Reply:
but here the file size is so small that it's quite useless

Reply:
Even now we use `cp` https://github.com/eclipse/che/blob/master/dockerfiles/che/build.sh#L31 )

Reply:
the reason is different. (to have moved from ln to cp) It was to do some very weird stuff